packages/terra-base/tests/nightwatch/NoTranslationsLoadedBase.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Base from '../../lib/Base';
+
+const BaseNoTranslationsLoaded = () => {
+  const noTranslations = (
+    <Base locale="gabcdef" translationsLoadingPlaceholder={<div>No Translations</div>}>
```

Comment:
The night watch test seems to pass.  I couldn't find a way to test this in night watch, other than to make translations completely not go through.  Which meant fail loading.  I'm cool with dropping night watch if we don't want that.